homeassistant/components/utility_meter/sensor.py

```diff
@@ -36,6 +36,7 @@
     DATA_UTILITY,
     HOURLY,
     MONTHLY,
+    NONE,
```

Comment:
unused...

Reply:
So what do you suggest then? I think there should always be a cycle specified by the user so we need a new constant "NONE". Making the `cycle`optional is not a good idea in my opinion.
But the UM does not need to do anything with it, that's why I added the "pass" in the previous version of the code (also to make sure the tests don't throw a "not used" error.
In the previous revieuw you stated that I should not worry about the tests not passing and so I removed the `pass`section in the code.
Seems like we're running in circles here.

Reply:
in this file you are just importing the variable which is not used...

Do you program in python :) ?

Reply:
I'm by far not a python expert - I have a few degrees in C++ and Java, but the discussion is not relevant to the coding language.

I think that adding an option to UM that enables the user to have a UM that does not reset automatically has significant value.
There are 3 options:

1. Make `cycle` an optional parameter in the integration
2. Import the parameter and when `async_added_to_hass`runs, make sure no `async_track_time_change` is created (what I did in the previous version by adding `pass`)
3. Import the parameter and do not use it

If you have any better suggestion(s), please let me know - but now I've switched from 2 to 3 based on your feedback, and now 3 is not ok.

Reply:
We are probably getting lost in translation 

I'm not discussing the need for the `"none"` cycle, it has come up in the past in user forums the suggestion to add it and I agree with it.

You only need `NONE` to be defined in the `consts.py` in order for the configuration validation to pass.

BTW: we are very short of implementing option 1. (just a matter of defining `none` as the default cycle) and doing the proper checks